lib/jmap/core/error/method/exception/error_method_response_exception.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/core/method/method_response.dart';
+
+class ErrorMethodResponseException with EquatableMixin implements Exception {
+  final MethodResponse errorResponse;
+
+  ErrorMethodResponseException(this.errorResponse);
+
+  @override
+  List<Object?> get props => [errorResponse];
+
+  @override
+  bool? get stringify => true;
```

Comment:
why need it?

Reply:
it will be useful when using `toString` for the instance of this class. More readable